packages/integration-tests/controllerManager.js

```diff
@@ -0,0 +1,134 @@
+module.exports = (controller, firmwares) => ({
```

Comment:
nice, sort of future trezor-user-env-link API here. could you please somehow incorporate this file into `packages/trezor-user-env-link` ? Or maybe we could do  this later, no problem